aten/src/ATen/functorch/LegacyVmapTransforms.cpp

```diff
@@ -102,7 +102,7 @@ static Tensor moveDimToFrontAndExpand(Tensor tensor, std::optional<int64_t> dim,
   } else {
     tensor = tensor.unsqueeze(0);
     auto expanded_sizes = tensor.sym_sizes().vec();
-    expanded_sizes[0] = size;
+    expanded_sizes[0] = std::move(size);
     tensor = tensor.expand_symint(expanded_sizes);
```

Comment:
```suggestion
    tensor = tensor.expand_symint(std::move(expanded_sizes));
```
I think this take it by value, but I could be wrong.

Reply:
The move is useless because the moved to parameter is array_ref.